src/QafooLabs/Bundle/NoFrameworkBundle/Tests/Controller/ResultConverter/ArrayToTemplateResponseConverterTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace QafooLabs\Bundle\NoFrameworkBundle\Tests\Controller\ResultConverter;
+
+use PHPUnit\Framework\TestCase;
+
+use QafooLabs\Bundle\NoFrameworkBundle\Controller\ResultConverter\ArrayToTemplateResponseConverter;
+use Symfony\Component\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use QafooLabs\Bundle\NoFrameworkBundle\View\TemplateGuesser;
+use QafooLabs\MVC\TemplateView;
+
+class ArrayToTemplateResponseConverterTest extends TestCase
+{
+    private $templating;
+    private $guesser;
+    private $converter;
+
+    public function setUp()
+    {
+        $this->converter = new ArrayToTemplateResponseConverter(
+            $this->templating = \Phake::mock(EngineInterface::class),
+            $this->guesser = \Phake::mock(TemplateGuesser::class),
+            'twig'
+        );
+    }
+
+    public function testSupports()
```

Comment:
What do you think about adding a return type declaration here, and on all other test methods?